src/main/java/de/zalando/aruha/nakadi/config/RepositoriesConfig.java

```diff
@@ -26,8 +32,19 @@
     @Autowired
     private JdbcTemplate jdbcTemplate;
 
+    @Autowired
+    private ZookeeperConfig zookeeperConfig;
+
     @Bean
-    public EventTypeRepository eventTypeRepository() {
-        return new EventTypeDbRepository(jdbcTemplate, jsonConfig.jacksonObjectMapper());
+    public EventTypeRepository eventTypeRepository() throws Exception {
+        final EventTypeRepository dbRepo = new EventTypeDbRepository(jdbcTemplate, jsonConfig.jacksonObjectMapper());
+
+        final String connectString = zookeeperConfig.zooKeeperHolder().get().getZookeeperClient().getCurrentConnectionString();
+        final RetryPolicy retryPolicy = new ExponentialBackoffRetry(1000, 3);
+        final CuratorFramework client = CuratorFrameworkFactory.newClient(connectString, retryPolicy);
```

Comment:
You were right. Done here https://github.com/zalando/nakadi/commit/c3aa6f2c4079e436bf2834caa96f4bf737e27ccc